Workflow/Sources/SubtreeManager.swift

```diff
@@ -349,7 +349,12 @@ extension WorkflowNode.SubtreeManager {
                 handler(event)
 
             case .invalid:
-                fatalError("[\(WorkflowType.self)] Sink sent an action after it was invalidated. Sinks can only be used for a single valid `Rendering`.")
```

Comment:
yes i believe it's fairly uncommon, at least according to our existing bugsnag data. we don't actually currently have a straightforward mechanism to log errors/warnings to something listening from outside of this module, so that can be taken up as a follow-on task.